pkg/virtctl/create/vm_test.go

```diff
@@ -0,0 +1,438 @@
+package create_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strconv"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "kubevirt.io/api/core/v1"
+	instancetypeapi "kubevirt.io/api/instancetype"
+	"sigs.k8s.io/yaml"
+
+	"kubevirt.io/kubevirt/tests/clientcmd"
+
+	. "kubevirt.io/kubevirt/pkg/virtctl/create"
+)
+
+const (
+	cloudInitUserData = `#cloud-config
+user: user
+password: password
+chpasswd: { expire: False }`
+
+	cloudInitNetworkData = `network:
+  version: 1
+  config:
+  - type: physical
+  name: eth0
+  subnets:
+    - type: dhcp`
+)
+
+var _ = Describe("create vm", func() {
+	Context("Manifest is created successfully", func() {
+		It("VM with random name", func() {
+			out, err := runCmd()
+			Expect(err).ToNot(HaveOccurred())
+			_ = unmarshalVM(out)
+		})
+
+		It("VM with specified name", func() {
+			const name = "my-vm"
+			out, err := runCmd(setFlag(NameFlag, name))
+			Expect(err).ToNot(HaveOccurred())
+			vm := unmarshalVM(out)
+
+			Expect(vm.Name).To(Equal(name))
+		})
+
+		It("VM with specified termination grace period", func() {
+			const terminationGracePeriod int64 = 123
+			out, err := runCmd(setFlag("termination-grace-period", fmt.Sprint(terminationGracePeriod)))
+			Expect(err).ToNot(HaveOccurred())
+			vm := unmarshalVM(out)
+
+			Expect(*vm.Spec.Template.Spec.TerminationGracePeriodSeconds).To(Equal(terminationGracePeriod))
+		})
+
+		DescribeTable("VM set running to", func(running bool) {
+			out, err := runCmd(setFlag(RunningFlag, strconv.FormatBool(running)))
```

Comment:
Can you add a test for the default value of False?

Reply:
ACK